patroni/dcs/__init__.py

```diff
@@ -295,8 +295,10 @@ def state(self) -> str:
 
     @property
     def is_running(self) -> bool:
-        """``True`` if the member :attr:`~Member.state` is ``running``."""
-        return self.state == 'running'
+        """``True`` if the member :attr:`~Member.state` is :class:`PostgresqlState.RUNNING`."""
```

Comment:
I think it should be `:attr:` instead of `:class:`?
```suggestion
        """``True`` if the member :attr:`~Member.state` is :attr:`PostgresqlState.RUNNING`."""
```

Reply:
although something is still off here